src/gantry/screens.py

```
@@ -666,6 +675,55 @@ def action_focus_search(self) -> None:
+    def action_focus_filter(self) -> None:
+        """Show and focus the advanced filter panel."""
+        filter_panel: FilterPanel = self.query_one("#filter-panel", FilterPanel)
+        filter_panel.add_class("show")
+        self.current_panel = "filter"
+        try:
+            inp = filter_panel.query_one("#filter-expr-input", Input)
+            inp.focus()
+        except NoMatches:
+            filter_panel.focus()
+
+    def on_filter_panel_filter_dismissed(self, event: FilterPanel.FilterDismissed) -> None:
```

Ruff (0.15.12)

[warning] Unused method argument: `event`

(ARG002)